src/sst/core/configGraph.cc

```diff
@@ -30,6 +31,48 @@ namespace SST {
 
 // static bool zero_latency_warning = false;
 
+// Functions to check component and link names
+static int   bad_comp_name_count = 0;
+static int   bad_link_name_count = 0;
+static regex valid_name_regex =
```

Comment:
All these statics can just be wrapped in an anonymous namespace and save on putting `static` in front of them. It doesn't change behavior, just how you write it.